src/unk_02054884.c

```diff
@@ -175,41 +175,41 @@ void Party_GiveChampionRibbons(Party *party)
     }
 }
 
-int sub_02054B04(Party *param0, u16 param1)
+int Pokemon_DoPoisonDamage(Party *party, u16 param1)
```

Comment:
**suggestion:** `param1` here looks like `mapLabelTextID`, looking at the invocation in `Field_UpdatePoison()` as well as what it eventually gets compared to (met location) in `Pokemon_UpdateFriendship()`.

Reply:
just wondering in general, is there a difference between a `mapId` and a `mapLabelTextId`?

Reply:
I personally prefer `ID`, but it's not a huge deal in either direction

Reply:
I believe `mapID` is the unique numeric identifier for the map (so basically what it sounds like), while `mapLabelTextID` refers to the index of the text entry that corresponds to its name in `message_bank_location_names.gmm`.

I agree with Lhea, I prefer and have been using `ID` rather than `Id`.